src/utils/Logging.ts

```diff
@@ -719,6 +721,9 @@ export default class Logging {
       for (const sensitiveData of Constants.SENSITIVE_DATA) {
         // Anonymize
         message.replace(new RegExp(sensitiveData, 'gi'), Constants.ANONYMIZED_VALUE);
+        // Should be
+        // message=message.replace(new RegExp(sensitiveData, 'gi'), Constants.ANONYMIZED_VALUE); //in order to anonymize the FULL string
```

Comment:
`message.replace(new RegExp(sensitiveData, 'gi') ... = message.replace(new RegExp(sensitiveData, 'gi') ... `
The fix proposal does not fix anything ;) 
If you want to fix the regexp, you should look for the string after the senvisite data key and obfuscated only that part: 
+ password: blah blah -> password: ####
+ password=blah blah -> password=###
+ ...

Reply:
Will work on this, thanks. Is it ok if I do it in this PR? Since it's not related to tests.

Reply:
> Will work on this, thanks. Is it ok if I do it in this PR? Since it's not related to tests.

That's fine. Update the PR changelog accordingly.

Reply:
Needless to say having sensitive data that might not be properly obfuscated is a security issue that have a very high priority and must be fixed ASAP ;)

Reply:
I know, it is my main focus. :)